FlowManager/displayManager.m

```diff
@@ -623,8 +623,20 @@ function resetPropQCCallback()
           
           for j=1:length(resetIdx)
               for i=1:length(sample_data{resetIdx(j)}.variables)
-                  if isfield(sample_data{resetIdx(j)}.variables(i), 'ancillary_comment')
-                      sample_data{resetIdx(j)}.variables(i) = rmfield(sample_data{resetIdx(j)}.variables(i), 'ancillary_comment');
+                  % previous code could not run since it was using variables(i) instead of variables{i}.
+                  % TODO ask @ggalibert about this
+                  if isfield(sample_data{resetIdx(j)}.variables{i}, 'ancillary_comment')
```

Comment:
@ggalibert quick question about syntax. Previous code didn't seem to be able to run. Am I missing something re matlab syntax?

Reply:
Must have been a typo, if sample_data.variable is a cell array then curly bracket should be used. Makes sense it is a cell array since not all variables structures have the same fields.